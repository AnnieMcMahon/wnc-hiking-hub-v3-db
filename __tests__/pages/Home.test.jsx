import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import Home from "@/app/page";

describe("Home", () => {
  it("matches the snapshot", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });

  it("renders a title", () => {
    render(<Home />);
    expect(screen.getByText(/How to Use This Site/)).toBeInTheDocument();
  });
});


