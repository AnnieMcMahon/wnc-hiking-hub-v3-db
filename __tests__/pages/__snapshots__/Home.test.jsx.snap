// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home matches the snapshot 1`] = `
<div>
  <div
    class="text-box w-[95%] md:w-120 my-4 mx-auto py-4 px-1 md:px-8 leading-8"
    id="home"
  >
    <h1>
      How to Use This Site
    </h1>
    <ul
      class="m-0 md:m-2"
    >
      <li>
        Create a free account using the 
        <b>
          Log In / Sign Up
        </b>
         button
      </li>
      <li>
        Edit your 
        <b>
          Bio
        </b>
         page and add a picture and description
      </li>
      <li>
        <b>
          Post a hike
        </b>
         you would like others to explore with you
      </li>
      <li>
        <b>
          Join a hike
        </b>
         someone else posted
      </li>
      <li>
        Go to your 
        <b>
          Bio
        </b>
         page to view or edit your hikes
      </li>
      <p
        class="text-center mt-2"
      >
        Happy hiking!
      </p>
    </ul>
  </div>
</div>
`;
